packages/react/src/components/error-summary/error-summary.tsx

```diff
@@ -22,6 +22,10 @@ const ErrorItem = styled.li`
     margin-bottom: var(--spacing-1x);
 `;
 
+const StyledSectionalBanner = styled(SectionalBanner)`
+    padding-bottom: var(--spacing-3x);
```

Comment:
This is not necessary. With adjustments on the `ul`, the space at the bottom matches the design.

Reply:
done